clients/bash/extract_and_check_a_single_bam.sh

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env bash
+
+: '
+Use the fingerprint service id to complete the following steps:
+1. Extract the fingerprint from the bam file path
+2. Then check the fingerprint against all other known fingerprints
+
+Fingerprint uses the AWS Cloud Map service - https://ap-southeast-2.console.aws.amazon.com/cloudmap/home/namespaces
+to enable a user to find the service/instances of a cloud formation.
+
+We cannot hardcode our statemachine values in this script since they may update at short notice with a change to the cdk.
+'
+
+# Set to fail
+set -euo pipefail
+
+# Set globals
+SERVICE_NAME="fingerprint"
+BAM_GDS_FILE_PATH="gds://production/analysis_data/SBJ00005/wgs_alignment_qc/20211201bd0ac3a3/L2101368__4_dragen/PTC_Tsqn211109.bam"
```

Comment:
This is the only global that is meant to be altered by the user - whereas all the others are fundamental properties of the installation.. maybe don't call this one a global.. or separate it from the other definitions.

Reply:
I'd maybe put it right at the top even before the first comment..

Reply:
I've updated it to use a keywarg input parameter instead